server/Models/getRestaurantInfo.js

```diff
@@ -0,0 +1,68 @@
+const pg = require('../db');
+
+const getRestaurantInfo = (req, res) => {
+  const { restaurantId } = req.params;
+
+  const { sort, keyword, star } = req.body;
+  let orderByStr = '';
+  let selectStr = orderByStr;
+  let sortDirection = '';
+  let keywordStr = '';
+  let starStr = '';
+
+  if (sort === 'highest rating') {
+    orderByStr = 'CAST((reviews.foodrating+reviews.servicerating+reviews.ambiencerating+reviews.valuerating) AS FLOAT)/4';
+    sortDirection = ' DESC,';
+  }
+
+  if (sort === 'lowest rating') {
+    orderByStr = 'CAST((reviews.foodrating+reviews.servicerating+reviews.ambiencerating+reviews.valuerating) AS FLOAT)/4';
+    sortDirection = ' ASC,';
+  }
+
+  if (keyword !== undefined) {
+    for (let i = 0; i < keyword.length; i ++) {
+      keywordStr += `AND reviews.reviewText LIKE '%${keyword[i]}%' `;
+    }
+
+    keywordStr = keywordStr.slice(0, -1);
```

Comment:
is this for removing the extra space at the end?

Reply:
The original purpose of this line was to remove the last comma and the space of the `keywordStr`. Now switched to `keywordStr = keywordStr.trim();`. Changes will be reflected in the next merge.